### DISCUSSION TITLE
Make page rollback possible from the page's Settings panel

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1933] [Ideas]
Having more than one Sign-in block on a page will crash the page, and this was submitted as a bug (https://github.com/vanjarosoftware/Vanjaro.Platform/issues/1932).

This bug illustrates how a page can crash while the display is being rendered, and when it does, it also prevents the Vanjaro panel to load. The problem now arises that there is no way to access the page's settings to rollback the page.

This suggestion is to include the Revisions (Page Revisions) display - which includes the ability to rollback - in the existing Page Settings panel. That way, the page could easily be rolled back by going to the Site / Pages area and accessing the crashed page's settings (and the suggested Page Revisions control) to easily rollback the crashed page. Since that presently isn't possible, the only way to rollback the page is to delete the version directly in the database.

Here's a screenshot of what I'm talking about:

![2021-11-28_6-40-17](https://user-images.githubusercontent.com/8103212/143767743-f0e4db2f-f666-47be-96bc-b5447d5a2564.jpg)

**Comment by meetmandeep** [comment]
Great idea. Moving it to issues.
